config/jobs/kubernetes/sig-scalability/DRA/sig-scalability-presubmit-dra-capz.yaml

```diff
@@ -168,9 +168,9 @@ presubmits:
         - name: CONTROL_PLANE_MACHINE_TYPE
```

Comment:
I am not able to comment but on Line 154, we need `--nodes=1000`

Reply:
lol we're declaring `--nodes` twice